frontend/src/css/modules/challenge-host-team.scss

```diff
@@ -0,0 +1,5 @@
+/* My existing host team card styles */
+
+.btn-create {
```

Comment:
This css is for the `Next` button on Select challenge host team card. We have `.list-icon` class in `base.scss` file but it is commented. I didn't un-comment it as it can cause impact on other elements too. Please guide what can be best approach here ?